spec/models/notification_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Notification, type: :model do
+  let!(:notification) { create(:notification) }
+
+  describe 'factory specs' do
+    it 'has factory' do
+      expect(notification).to be_persisted
```

Comment:
no use, factory girl raises error if it cannot create a record
replace w/ `build` + `be_valid`